code/modules/assembly/flash.dm

```diff
@@ -311,3 +312,60 @@
 /obj/item/assembly/flash/shield/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
 	activate()
 	return ..()
+	
+/obj/item/assembly/flash/hypnotic
+	desc = "A modified flash device, programmed to emit a sequence of subliminal flashes that can send a vulnerable target into a hypnotic trance."
+	flashing_overlay = "flash-hypno"
+	light_color = LIGHT_COLOR_PINK
+	cooldown = 20
+	
+/obj/item/assembly/flash/hypnotic/burn_out()
+	return
+
+/obj/item/assembly/flash/hypnotic/proc/check_vulnerable(mob/living/carbon/C)

```

Comment:
Is this leftovers from WIP? It's never called and it looks like its role is taken by carbon's hypnosis_vulnerable()

Reply:
Oh i forgot to delete tgat yeah